src/modules/inventory/src/inventory.cpp

```diff
@@ -0,0 +1,155 @@
+/*
```

Comment:
I think we can leave this comment for another PR as many files still don't have it.